### PR TITLE
refactor!: Project abstraction

### PR DESCRIPTION
**Pull request by JigarJoshi**
- create/drop database --> create/drop project.
- list databases --> list projects.

### CODE REVIEW

File: cmd/database.go [hunk 1]
```diff
@@ -26,16 +26,15 @@ import (
 
 var schemaOnly bool
 
-var listDatabasesCmd = &cobra.Command{
-	Use:     "databases",
-	Aliases: []string{"projects"},
-	Short:   "Lists databases/projects",
-	Long:    "Project is an alias for Database - this command will list all projects/databases",
+var listProjectsCmd = &cobra.Command{
+	Use:   "projects",
```
Comment (efirs) [inline cmd/database.go:30]: Can we add 'database(s)' alias to the commands, so it's become backward compatible change?

Comment (JigarJoshi) [reply cmd/database.go:30]: It was like this (projects) was an alias. We discussed and decided to make the cut over to project completely.

Comment (efirs) [reply cmd/database.go:30]: Then it'll break few places where CLI is used.
CLI integration tests, YCSB, backups maybe.

Comment (JigarJoshi) [reply cmd/database.go:30]: Yes it is breaking change.

**Review by efirs — APPROVED** [review]

**Pull request merged**